test/test-lib-ruby.sh

```diff
@@ -37,10 +37,9 @@ function rails_ex_branch() {
   # that are not compatible with RHEL 7.
 
   # Latest stable
-  rails_example_repo_branch="3.3"
-  if { echo "$VERSION"; echo "3.1"; } | sort --version-sort --check=quiet; then
-    # Ruby 3.1 and prior.
-    rails_example_repo_branch="master"
+  rails_example_repo_branch="master"
+  if [ "$VERSION" == "3.3" ] || [ "$VERSION" == "3.1" ]; then
+    rails_example_repo_branch="3.3"
   fi
```

Comment:
The previous behavior was targeted so that 3.1 also uses the "master" branch IIRC:
```console
$ export VERSION=3.3; { echo "$VERSION"; echo "3.1"; } | sort --version-sort --check=quiet; echo $?
1
$ export VERSION=3.1; { echo "$VERSION"; echo "3.1"; } | sort --version-sort --check=quiet; echo $?
0
```

Reply:
I see, ok, I will revert it. And let's see how test are working.